packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -151,6 +151,14 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
     thread?.topic?.id,
   );
 
+  useEffect(() => {
```

Comment:
we should check if the user is a thread admin before updating this state. A thread admin is anyone who has any of these 4 permissions

https://github.com/hicommonwealth/commonwealth/blob/06a3b518b00011d7149a63ea55b0732855210089/packages/commonwealth/client/scripts/views/pages/discussions/ThreadCard/ThreadOptions/AdminActions/AdminActions.tsx#L74-L79

Reply:
if user  is not thread admin he can not navigate to this page , there will be no dropdown for edit thread 
![image](https://github.com/user-attachments/assets/00fc4425-358a-43ef-910f-a352d270e73c)
@mzparacha

Reply:
they can always directly update the url with `?isEdit=true`

Reply:
@mzparacha  i added the extra check as well and i test it as well